balboa-client-kafka/src/main/scala/com/socrata/metrics/impl/MetricLoggerToKafka.scala

```diff
@@ -0,0 +1,100 @@
+package com.socrata.metrics.impl
+
+import com.socrata.balboa.metrics.Metric.RecordType
+import com.socrata.balboa.common.kafka.util.AddressAndPort
+import com.socrata.metrics.collection.LinkedBlockingPreBufferQueue
+import com.socrata.metrics.components.{MetricEnqueuer, MetricEntry, MetricLoggerComponent}
+import org.apache.commons.logging.LogFactory
+
+/**
+ * Information that pertains to how to setup and configure producers that will communicate with Kafka.
+ */
+trait KafkaProducerInformation {
+
+  /**
+   * @return List of Kafka Brokers that exists within this environment.
+   */
+  def brokers: List[AddressAndPort]
+
+  /**
+   * @return The name of the topic that is being used.
+   */
+  def topic: String
+
+  /**
+   * @return The back up file where failed writes will go.
+   */
+  def backupFile: String
```

Comment:
I think I would rename this to better reflect that this is for emergency backups. "backupFile" implies that this is a normal condition; but I would go with "emergencyBackupFile" or something to make it clear.